javascript/javascript1/week2/formalFullname.js

```diff
@@ -0,0 +1,36 @@
+function getFullname(firstname, surname, useFormalName) {
+  let message;
+  if (!firstname || !surname) {
+    console.log("Firstname and Surname fields should be filled!");
+    return;
+  }
+
+  if (useFormalName) {
+    message = `Lord ${firstname} ${surname}`;
+  } else {
+    message = `${firstname} ${surname}`;
+  }
+
+  return message;
```

Comment:
Here you can just return early.

```suggestion
  if (useFormalName) {
    return `Lord ${firstname} ${surname}`;
  } else {
    return `${firstname} ${surname}`;
  }
```

This would work. To make it even shorter and clean you can do 

```suggestion
  if (useFormalName) {
   return `Lord ${firstname} ${surname}`;
  }
   
    return `${firstname} ${surname}`;

```